dts/bindings/sensor/microchip,mcp9808.yaml

```diff
@@ -18,3 +18,14 @@ properties:
         sensor, and is open-drain.  A pull-up may be appropriate.  The
         property value should ensure the flags properly describe the
         signal that is presented to the driver.
+
+    resolution:
+      type: int
+      required: false
+      default: 3
```

Comment:
Is there a reason you picked this default value? Is it the reset value in the sensor register? Please document where the default came from in the description.

Reply:
This is the default, power-on, value. This information has now been documented.